Analytics/Classes/Internal/SEGHTTPClient.h

```diff
@@ -5,10 +5,11 @@
 @interface SEGHTTPClient : NSObject
 
 @property (nonatomic, strong) SEGRequestFactory requestFactory;
+@property (nonatomic, copy, readwrite) NSString *endpoint;
```

Comment:
Is this var really the endpoint or is it actually host?  I'm thinking it should be called `host` here and in this usage that follows in the other files.  If you agree, can you make this change?

Otherwise LGTM.

Reply:
@tedmiston yeah, i'm on board with that. i went with endpoint because that is how it was specified in the open issue, but i like host better too. i'll get that changed.